src/test/miner_tests.cpp

```diff
@@ -284,6 +284,13 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
         BOOST_CHECK_MESSAGE(state.IsValid(), state.GetRejectReason());
         pblock->hashPrevBlock = pblock->GetHash();
     }
+
+    // Set the clock to be just ahead of the last "mined" block, to ensure we satisfy the
+    // future timestamp soft fork rule.
```

Comment:
I think it might have ended up in the protocol spec? The security notice for its introduction is here: https://z.cash/support/security/announcements/security-announcement-2020-02-06/

Reply:
This was specified in protocol spec revision 2020.1.0.